lib/ansible/modules/network/avi/avi_api_session.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+"""
+# Created on Aug 12, 2016
+#
+# @author: Gaurav Rastogi (grastogi@avinetworks.com) GitHub ID: grastogi23
+#
+# module_check: not supported
```

Comment:
What would be needed to support check mode? Idempotent modules is one of Ansible's key features